lib/foreman_discovery/engine.rb

```diff
@@ -88,7 +89,8 @@ class Engine < ::Rails::Engine
               setting "discovery_prefix",
                 type: :string,
                 default: "mac",
-                validate: { presence: true, format: { with: /\A[a-zA-Z]/, message: _("must start with a letter") } },
+                validate: { presence: true, format: { with: /\A([A-Za-z][A-Za-z0-9-]{0,61})\z/,
+                            message: _("must start with a letter. Allowed characters: `aA-zZ`, `0-9` and `-`. Max length is 62 characters.") } },
```

Comment:
```suggestion
                            message: _("must start with a letter. Allowed characters: `a-z`, `A-Z`, `0-9` and `-`. Max length is 62 characters.") } },
```

I'd explicitly call them out, `aA-zZ` looks weird